### ISSUE TITLE
InsertBatch temp table case insensitive column names cases

### ISSUE THREAD

**Issue by rchybicki** [cmeeren/Facil#57]
Hello,

The insertBatch script generated for this table:
```
CREATE TABLE [dbo].[zzz](
	[PRCo] [tinyint] NOT NULL,
	[Employee] [int] NOT NULL,
	[Document] [varchar](255) NOT NULL,
	....
) ON [PRIMARY]
GO
```

looks like this:
```
    let configureCmd sqlParams (cmd: SqlCommand) =
      cmd.CommandText <- """-- zzz
INSERT INTO [dbo].[zzz]
(
  [PRCo],
  [Employee],
  [Document],
....
)
SELECT
  [pRCo],
  [employee],
  [document],
...
FROM #args"""
```

While the CreateTempTable code is:
```
    [<EditorBrowsable(EditorBrowsableState.Never)>]
    member this.CreateTempTableData
      (
        ``args``: seq<``zzz``.``args``>
      ) =
      [
        TempTableData
          (
            "#args",
            """
            CREATE TABLE #args (
              [PRCo] TINYINT NOT NULL,
              [Employee] INT NOT NULL,
              [Document] VARCHAR(255) COLLATE Latin1_General_BIN NOT NULL,
              ...
            )
            """,
            (``args`` |> Seq.map (fun x -> x.Fields)),
            12,
            Action<_> this.userConfigureBulkCopy
          )
      ]
```

When this is executed on a case sensitive sql server, it doesn't work. The case of the column names should be preserved I believe, or am I doing something wrong? The collation of the DB that Facil is generating from is case sensitive, though the SQL Server's collation of that DB's server is not.

**Comment by cmeeren** [comment]
I believe you're right. I'll investigate.

**Comment by cmeeren** [comment]
Good catch! Should be fixed in 2.7.3 which is in the pipeline now.

**Comment by rchybicki** [comment]
Nice! I appreciate the super-fast reaction.
